rgw/v2/utils/utils.py

```diff
@@ -42,6 +42,17 @@ def exec_shell_cmd(cmd, debug_info=False):
     except Exception as e:
         log.error("cmd execution failed")
         log.error(e)
+        # dump the crash log information on to the console, if any
+        ceph_version_id, ceph_version_name = get_ceph_version()
```

Comment:
Can you use `_` instead of `ceph_version_id` since we are not using that variable further in this method

Reply:
@udaysk23 , sure made the changes